label_studio/core/utils/io.py

```diff
@@ -213,3 +215,15 @@ def validate_ip(ip: str) -> None:
     for subnet in local_subnets:
         if ipaddress.ip_address(ip) in ipaddress.ip_network(subnet):
             raise InvalidUploadUrlError
+
+
+def ssrf_safe_get(*args, **kwargs):
+    # Reason for #nosec: url has been validated as SSRF safe by the
+    # validation check above.
```

Comment:
nit: this comment is a little inaccurate now, probably worth clarifying. Could we call validate_upload_url in this function too?

Reply:
Fixed.